docs/source/benchmarking/materials_science_and_engineering/oiecharacterisation.rst

```diff
@@ -0,0 +1,68 @@
+Open Innovation Environment (OIE) domain ontologies, Characterisation module (OIECharacterisation)
```

Comment:
convert `Open Innovation Environment (OIE) domain ontologies, Characterisation module (OIECharacterisation)` to `Open Innovation Environment Characterisation (OIECharacterisation)`